aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -1184,7 +1184,7 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   TRIVIAL_OP(conj);
   TRIVIAL_OP(_conj);
   TRIVIAL_OP(resolve_conj);
-  TRIVIAL_OP(_resolve_conj);
+  TRIVIAL_OP(_resolve_conj_neg);
```

Comment:
Although it "doesn't really matter" because this is a private underscored function, it is weird naming, especially if you imagine us ever adding other view bits the name would keep getting longer and longer.

Reply:
This sort of relates to a discussion we had earlier about whether or not new/like should propagate conjugate/negative bits. Anything that *doesn't* get propagated gets "cleared" by the resolve method implemented by allocating a new tensor and copy_'ing into it. Is this the semantics of `clone()`? Here is clone:

```
Tensor clone(const Tensor& src, c10::optional<c10::MemoryFormat> optional_memory_format) {
  auto memory_format =
      optional_memory_format.value_or(MemoryFormat::Preserve);
  if (memory_format == MemoryFormat::Preserve) {
    if (src.is_non_overlapping_and_dense()) {
      // Copy all strides
      auto self = at::empty_strided(src.sizes(), src.strides(), src.options());
      self.copy_(src);
      return self;
    } else {
      memory_format = src.suggest_memory_format();
    }
  }
  auto self = at::empty_like(src, src.options(), memory_format);
  self.copy_(src);
  return self;
}
```

so really maybe just deferring to `clone` is sufficient.

Reply:
yeah good point! we can just use clone here